### PR TITLE
Prepare for Jenkins CI Build

### PR DESCRIPTION
**Pull request by qwq233**
# Prepare for Jenkins CI Build

## 更改内容 / Types of Changes

- build.gradle

## 由这个PR解决或关闭的Issues / Issues Fixed or Closed by This PR

-  

## 检查列表 / Checklist

- [x] 我的代码符合本仓库的代码规范 My code follows the code style of this project
- [x] 我已经测试了这些更改，它们可以正常工作，既不会破坏原有任何功能(或我可以解决)，也不会影响对旧版本的支持 I have tested the changes and verified that they work and don't break anything(as well as I can manage) or drop the support for previous versions.
- [x] 我已合并了对后续工作无意义的commit并确认不会对后续维护造成困扰 I have merged commits that are meaningless for follow-up work and confirmed that they will not cause damage to follow-up maintenance

### CODE REVIEW

**Comment by singleNeuron** [comment]
@qwq233 不加skip ci?

**Comment by qwq233** [comment]
@singleNeuron 加p呢 又循环了等下

**Comment by singleNeuron** [comment]
@qwq233 克制的加不是不加啊

File: app/build.gradle [hunk 1]
```diff
@@ -2,11 +2,28 @@ apply plugin: 'com.android.application'
...
+
 android {
     compileSdkVersion 29
     buildToolsVersion "29.0.3"
     ndkVersion "21.3.6528147"
-    
+    if (performSigning) {
+        signingConfigs {
+            release {
+                storeFile file(project.signing.storeFilePath)
```
Comment (singleNeuron) [inline app/build.gradle:18]: 完全没必要，你又不能把这个文件传进仓库，CI还是拿不到，不如用环境变量和CI的自带流程来签名了

**Review by singleNeuron — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (qwq233) [reply app/build.gradle:18]: jenkins可以provide config

**Review by singleNeuron — APPROVED** [review]

**Review by cinit — APPROVED** [review]

**Pull request merged**